db/queries/assessment_records/queries.py

```diff
@@ -105,3 +108,44 @@ def find_answer_by_key_runner(field_key: str, app_id: str) -> List[tuple]:
         .filter(AssessmentRecord.application_id == app_id)
         .one()
     )
+
+
+def find_assessor_task_list_state(application_id: str) -> dict:
```

Comment:
Might we want to return tasklist specific state here. For example, we only want criteria scoring and statuses, so not sure if we need the json blob of answers.

Reply:
I think so too yeah.  Will need to find out when that work can actually be done, as I think there are in-flight stories at the moment making changes to the data model?  I'll actually ask Gio to get confirmation on this, because if not I might be able to do TDD on the current data model

Reply:
Have created https://digital.dclg.gov.uk/jira/browse/FS-2000 for when the `score` data model changes are in place and that work can be picked up 👍

Reply:
Have created https://digital.dclg.gov.uk/jira/browse/FS-2000 for when the score data model changes are in place and that work can be picked up 👍